plugins/gpg-agent/gpg-agent.plugin.zsh

```diff
@@ -15,7 +15,15 @@ function start_agent_withssh {
 }
 
 # check if another agent is running
-if ! gpg-connect-agent --quiet /bye > /dev/null 2> /dev/null; then
+if gpg-connect-agent --quiet /bye > /dev/null 2> /dev/null; then
+    # source settings of old agent, if applicable
+    if [ -f "${GPG_ENV}" ] && [ -z "${SSH_AGENT_PID}"]; then
```

Comment:
`.oh-my-zsh/plugins/gpg-agent/gpg-agent.plugin.zsh:[:20: ']' expected`

lack of space after `"${SSH_AGENT_PID}"`